packages/tasks/src/local-apps.ts

```diff
@@ -143,17 +148,29 @@ const snippetLocalAI = (model: ModelData, filepath?: string): LocalAppSnippet[]
 };
 
 const snippetVllm = (model: ModelData): LocalAppSnippet[] => {
-	const runCommand = [
+	const runCommandInstruct = [
 		"# Call the server using curl:",
-		`curl -X POST "http://localhost:8000/v1/chat/completions" \\ `,
-		`	-H "Content-Type: application/json" \\ `,
+		`curl -X POST "http://localhost:8000/v1/chat/completions" \\`,
+		`	-H "Content-Type: application/json" \\`,
 		`	--data '{`,
 		`		"model": "${model.id}",`,
 		`		"messages": [`,
 		`			{"role": "user", "content": "Hello!"}`,
```

Comment:
```suggestion
		`			{"role": "user", "content": "What is the capital of France?"}`,
```

Minor suggestion: `Hello!` looks a bit too terse. Perhaps we can unify the Instruct examples to be the same as llama-cpp-python and so on.

Reply:
handled in https://github.com/huggingface/huggingface.js/pull/954/commits/2e7c0808ca75a10222d4fc9bcdda1fff7660aa19